tests/shutdown/cleanup_before_shutdown.pm

```diff
@@ -25,6 +25,7 @@ use testapi;
 use serial_terminal 'prepare_serial_console';
 use utils;
 use version_utils;
+use Utils::Backends 'is_qemu';
 
 sub run {
     select_console('root-console');
```

Comment:
while you're at this, maybe switch to the serial console?

Reply:
rather not to touch something which is not utterly broken to avoid unexpected consequences